src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -378,8 +378,17 @@ void Pane::_ControlWarningBellHandler(const winrt::Windows::Foundation::IInspect
                 PlaySound(soundAlias, NULL, SND_ALIAS_ID | SND_ASYNC | SND_SENTRY);
             }
 
-            // raise the event with the bool value corresponding to the visual flag
-            _PaneRaiseBellHandlers(nullptr, WI_IsFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Visual));
+            if (WI_IsFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Visual) ||
+                WI_IsFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Window))
+            {
+                _control.InvertScreenColors();
+            }
+
+            const auto flashTaskbar = WI_IsFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Visual) ||
+                                      WI_IsFlagSet(paneProfile.BellStyle(), winrt::Microsoft::Terminal::Settings::Model::BellStyle::Taskbar);
+
+            // raise the event with the bool value corresponding to the taskbar or visual flag
+            _PaneRaiseBellHandlers(nullptr, flashTaskbar);

```

Comment:
So, the settings now look like this?
- `window`: flash terminal
- `taskbar`: add bell icon to tab
- `visual` = `[ "window", "taskbar" ]`

Reply:
`taskbar` flashes the taskbar icon at the toolbar (if the terminal window is not focused)! The others are correct